PS C:\Users\reddyvu\Desktop\smartdq\dq_api> pip install teradatasql
Requirement already satisfied: teradatasql in c:\users\reddyvu\appdata\local\programs\python\python312\lib\site-packages (20.0.0.15)
Requirement already satisfied: pycryptodome in c:\users\reddyvu\appdata\local\programs\python\python312\lib\site-packages (from teradatasql) (3.20.0)
PS C:\Users\reddyvu\Desktop\smartdq\dq_api> pip install teradatasqlalchemy
WARNING: Retrying (Retry(total=4, connect=None, read=None, redirect=None, status=None)) after connection broken by 'ProtocolError('Connection aborted.', ConnectionResetError(10054, 'An existing connection was forcibly closed by the remote host', None, 10054, None))': /simple/teradatasqlalchemy/        
WARNING: Retrying (Retry(total=3, connect=None, read=None, redirect=None, status=None)) after connection broken by 'ProtocolError('Connection aborted.', ConnectionResetError(10054, 'An existing connection was forcibly closed by the remote host', None, 10054, None))': /simple/teradatasqlalchemy/        
WARNING: Retrying (Retry(total=2, connect=None, read=None, redirect=None, status=None)) after connection broken by 'ProtocolError('Connection aborted.', ConnectionResetError(10054, 'An existing connection was forcibly closed by the remote host', None, 10054, None))': /simple/teradatasqlalchemy/        
WARNING: Retrying (Retry(total=1, connect=None, read=None, redirect=None, status=None)) after connection broken by 'ProtocolError('Connection aborted.', ConnectionResetError(10054, 'An existing connection was forcibly closed by the remote host', None, 10054, None))': /simple/teradatasqlalchemy/        
WARNING: Retrying (Retry(total=0, connect=None, read=None, redirect=None, status=None)) after connection broken by 'ProtocolError('Connection aborted.', ConnectionResetError(10054, 'An existing connection was forcibly closed by the remote host', None, 10054, None))': /simple/teradatasqlalchemy/        
ERROR: Could not find a version that satisfies the requirement teradatasqlalchemy (from versions: none)
ERROR: No matching distribution found for teradatasqlalchemy
PS C:\Users\reddyvu\Desktop\smartdq\dq_api> 
